afs/media/js/views/components/workflows/analysis-areas-annotation-step.js

```diff
@@ -174,11 +176,15 @@ define([
             self.selectAnalysisAreaInstance(newTile);
         };
 
-        // this.saveTile = function() {
-        //     self.tile.save().then(function(data) {
-        //         self.observationInstances(self.card.tiles())
-        //     });
-        // };
+        this.saveAnalysisArea = function () {
+            params.form.complete(false);
+            params.form.saving(true);
+            self.analysisAreaInstances().forEach(
+                instance => params.form.savedData.push({ "data": instance.data })
+            );
```

Comment:
nit: this can just be `params.form.savedData(self.analysisAreaInstances());`

Reply:
@chrabyrd this actually doesn't work; it throws an exception.

Reply:
(error converting circular structure to JSON)

Reply:
ah actually I got it wrong.

let's map the `data` attr from each `analysisAreaInstance` into an array.

Then, have `params.form.savedData` consume the mapped array.

That way we're only saving the data we care about ( `analysisAreaInstance.data`), we're avoiding the JSON error, and we're preventing unnecessary observer/subscription signals by only interacting with `params.form.savedData` once.